arbnode/batch_poster.go

```diff
@@ -117,6 +118,7 @@ type BatchPosterConfig struct {
 	DataPoster         dataposter.DataPosterConfig `koanf:"data-poster" reload:"hot"`
 	RedisUrl           string                      `koanf:"redis-url"`
 	RedisLock          redislock.SimpleCfg         `koanf:"redis-lock" reload:"hot"`
+	EnableRedisLock    bool                        `koanf:"enable-redis-lock"`
```

Comment:
this parameter seems better inside redislock.SimpleCfg

Reply:
Updated